app/scripts/controllers/network/network-controller.ts

```diff
@@ -742,10 +742,10 @@ export class NetworkController extends EventEmitter {
     const network = BUILT_IN_INFURA_NETWORKS[type];
     await this.#setProviderConfig({
       type,
-      rpcUrl: '',
```

Comment:
Nit: could we omit these properties instead of setting them to `undefined`? That seems closer to what we want to do, allowing us to improve the provider config type later on when we manage to enable exact optional property types.

Reply:
Yeah that would definitely be ideal. I was trying to match core on this but there's no reason why we couldn't delete these properties in this PR and then add a PR on the core side that removes them here. Would be easy changes either way.